applications/connectivity_bridge/src/modules/power_handler.c

```diff
@@ -41,15 +41,15 @@ static bool event_handler(const struct event_header *eh)
 		if (check_state(event, MODULE_ID(main), MODULE_STATE_READY)) {
 			module_active_count = 0;
 
-			k_delayed_work_init(&power_down_work, power_down_handler);
+			k_work_init_delayable(&power_down_work, power_down_handler);
```

Comment:
Calling init on a non-idle item can lead to invalid behavior. Preferably use `K_WORK_DELAYABLE_DEFINE` instead.

Reply:
Changing this